### PR TITLE
Fixed Typo's and Rephrased words

### PR DESCRIPTION
**Pull request by MrKrishnaAgarwal**
## What does this PR do?

I have fixed some minor typos and rephrased words

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes, I have read the [Contributing Guidelines](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)

cc @adityaoberai

### CODE REVIEW

**Review by MSamiDev — APPROVED** [review]

**Comment by gewenyu99** [comment]
🙏 While some of your suggestions are actually useful, we're generally not counting unsolicited grammar review PRs. These tend to be subjective and of lower quality/effort than we'd like. We may accept this PR, but we'll mark them invalid for Hacktoberfest. 

The best way to contribute during Hacktoberfest is to find an issue labeled `hacktoberfest` or to raise an issue and seek approval first. This makes it easier for us to track and avoid wasted time.

We love contributions, but we suggest you try and submit more meaningful PRs by exploring hacktoberfest issues via our [website](https://hacktoberfest.appwrite.io/). If you have any problems or need help, please join our [Discord community](https://appwrite.io/discord) and use the `#hacktoberfest` channel.

**Comment by MrKrishnaAgarwal** [comment]
Okay Sure @gewenyu99

Its okay if it doesn't count towards Hacktoberfest 😄

**Comment by MrKrishnaAgarwal** [comment]
You can still approve and merge it, no worries.

File: CONTRIBUTING.md
Comment (MrKrishnaAgarwal) [reply CONTRIBUTING.md:341]: It is there and then not and and then 😄

File: CONTRIBUTING.md
Comment (tessamero) [inline CONTRIBUTING.md:15]: this is good

File: CONTRIBUTING.md
Comment (tessamero) [inline CONTRIBUTING.md:33]: this is incorrect grammar

Comment (tessamero) [reply CONTRIBUTING.md:33]: "All PRs must include a commit message with the description of the changes made" would be accurate

**Review by tessamero — APPROVED** [review]

File: CONTRIBUTING.md
Comment (tessamero) [inline CONTRIBUTING.md:15]: why did you change this back? "follows" is correct

**Comment by MrKrishnaAgarwal** [comment]
done @tessamero

File: CONTRIBUTING.md
Comment (MrKrishnaAgarwal) [reply CONTRIBUTING.md]: resolved

**Comment by MrKrishnaAgarwal** [comment]
> Looks good! Thanks for the help <3

Welcome 😁

**Comment by MrKrishnaAgarwal** [comment]
@christyjacob4 and @gewenyu99 waiting for it to get merged and labelled as `hacktoberfest-accepted`.

**Comment by stnguyen90** [comment]
@MrKrishnaAgarwal, getting approved should be enough for Digital Oceans hacktoberfest event, but I've also added the hacktoberfest-accepted label now.

**Comment by MrKrishnaAgarwal** [comment]
> @MrKrishnaAgarwal, getting approved should be enough for Digital Oceans hacktoberfest event, but I've also added the hacktoberfest-accepted label now. 

Okay Thanks!

**Comment by MrKrishnaAgarwal** [comment]
> > @MrKrishnaAgarwal, getting approved should be enough for Digital Oceans hacktoberfest event, but I've also added the hacktoberfest-accepted label now. 
> 
> Okay Thanks!
I have aleardy completed Digital Ocean Hacktoberfest, I wanted to complete Appwrite Hacktoberfest

**Review by maharshi-sinha — APPROVED** [review]
Looks good to me! Thanks for your valuable contribution towards Appwrite

**Comment by MrKrishnaAgarwal** [comment]
> Looks good to me! Thanks for your valuable contribution towards Appwrite

Welcome :)
Looking forward for @christyjacob4 to merge the PR.

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Comment by MrKrishnaAgarwal** [comment]
> THANK YOU! All changes merged 🥳
> 
> Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  
> 
> You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.
> 
> You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.
> 
> Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)
> 

@christyjacob4 I didn't receive any invite for joining the org, please send again or check the issue 😕

**Pull request merged**
